### PR TITLE
Check in annotated data models for Charting page

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/docs/docs/charting/index.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
 
 ![Chart sample](/img/tutorials/chart-sample.png)
 
+## Data Model
+
```
Comment (reshmakh) [inline packages/docs/docs/charting/index.md:18]: Add some text here along the lines of: "All the elements you expect in a charting experience can be modeled in FHIR below are some examples of how the elements are represented

File: packages/docs/docs/charting/index.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
...
+![Chart Data Model 1](/img/tutorials/charting-annotation-1.png)
+
```
Comment (reshmakh) [inline packages/docs/docs/charting/index.md:20]: As the images don't contain hyperlinks, request to add a sentence below the chart.  You can read more about [Patient](/docs/api/fhir/resources/patient), [Appointment], [AllergyIntolerance]... in the reference material.

File: packages/docs/docs/charting/index.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
...
+![Chart Data Model 2](/img/tutorials/charting-annotation-2.png)
+
```
Comment (reshmakh) [inline packages/docs/docs/charting/index.md:22]: Similar to comment above, add sentence with links to resources below image.

File: packages/docs/docs/charting/index.md [hunk 4]
```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
...
+![Chart Data Model 3](/img/tutorials/charting-annotation-3.png)
+
```
Comment (reshmakh) [inline packages/docs/docs/charting/index.md:24]: Simlilar to above 2 comments here

File: packages/docs/docs/charting/index.md [hunk 5]
```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
 
 ![Chart sample](/img/tutorials/chart-sample.png)
 
+## Data Model
```
Comment (reshmakh) [inline packages/docs/docs/charting/index.md:17]: Possible to make these images a bit bigger?  Hard to read in current config

Comment (rahul1) [reply packages/docs/docs/charting/index.md:17]: Unfortunately the width is already maxed out. Do you think I should go back to Awesomic and ask them to increase the text size?

Comment (reshmakh) [reply packages/docs/docs/charting/index.md:17]: For the left-most column - I think yes.  Annotations are fine, could even be reduced a bit.  Send them the preview and they will get the picture I think

**Review by reshmakh — APPROVED** [review]

**Pull request merged**